Source/ConversationV1/Models/DialogNode.swift

```diff
@@ -44,28 +44,28 @@ public struct DialogNode {
     public var dialogNodeID: String
 
     /// The description of the dialog node.
-    public var description: String?
+    public var description: String
```

Comment:
There are a lot of `DialogNode` properties that the Swagger specified as `required` but were actually optional. Again, those changes are in the `staging/conversation` branch which is why they didn't show up in this generated code.